dashboard/test/lib/policies/child_account_test.rb

```diff
@@ -369,16 +369,16 @@ class ComplianceStateTest < ActiveSupport::TestCase
     context 'when the user does not have a country' do
```

Comment:
Can we also add a test case for `context 'when user is outside the US'` just for completeness?

Reply:
And or when `country_code` is `nil`

Reply:
Yes, good point. Will add it